scripts/style.sh

```diff
@@ -61,7 +61,7 @@ if [[ "$system" == "Darwin" ]]; then
   version="${version/*version /}"
   # Allow an older swiftformat because travis isn't running High Sierra yet
   # and the formula hasn't been updated in a while on Sierra :-/.
-  if [[ "$version" != "0.32.0" && "$version" != "0.33"* && "$version" != "0.35"* && "$version" != "0.37"* ]]; then
+  if [[ "$version" != "0.32.0" && "$version" != "0.33"* && "$version" != "0.35"* && "$version" != "0.36"* && "$version" != "0.37"* && "$version" != "0.38"* && "$version" != "0.39"* && "$version" != "0.4"* ]]; then
```

Comment:
So... what version does Travis have?

Initially there were just two versions because there was a lag between what we'd get on our desks and what was available to Travis. I think we slid down a (very) slippery slope when we started exempting more than the two.

What's the difference between this and no check at all?

Reply:
My machine has 0.35.7.  It seems that we're still not using enough Swift to trigger any differences from the versions.

I'm fine to stay flexible we're forced to get stricter.

Reply:
I said I'm not proud of this!  I just knew it would solve my problem. 😛 

How do we find out what version travis has?

Reply:
Add a commit with the command `swiftformat --version` and check the travis output.

Reply:
> YOUR SWIFT FORMAT VERSION IS: 0.35.7

Why are you shouting?